analysis_options.yaml

```diff
@@ -7,6 +7,8 @@ analyzer:
     todo: ignore
     unused_import: warning
     unused_shown_name: warning
+  language:
```

Comment:
This probably should be added to analysis_options_presubmit.yaml?

Reply:
Done.